rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -117,6 +136,16 @@ def to_exported_dict(
                 exported_dict['asset'] = ''
             if tx_hash is not None:
                 exported_dict['notes'] = f'{evm_explorer}{tx_hash}  ->  {self.notes}'
+            if self.location in EVM_LOCATIONS and database is not None:
+                # call _maybe_add_label_with_address on each address in the note
+                exported_dict['notes'] = re.sub(
+                    pattern=EVM_ADDRESS_REGEX,
```

Comment:
You are not using the compiled regex. Please when I say read the docs and look at the rest of the codebase do so

```suggestion
                exported_dict['notes'] = EVM_ADDRESS_REGEX.sub(
```